### ISSUE TITLE
Image Sliders

### ISSUE THREAD

**Issue by kurisubrooks** [kurisubrooks/caramel#19]
pls talons

**Comment by whoiskenjackson** [comment]
Let me know if you need any help with this. I have built a billion of these.

**Comment by kurisubrooks** [comment]
Sky's been busy with other things, but if you want to give it a shot, feel free!

**Comment by whoiskenjackson** [comment]
Do you have any links to an example? Carousel from Bootstrap? http://getbootstrap.com/javascript/#carousel

**Comment by kurisubrooks** [comment]
Yeah, something like that! But no awful gradient on the side buttons.

**Comment by whoiskenjackson** [comment]
Hahaha... Totally. Sweet! I'll take a stab at it.

**Comment by kurisubrooks** [comment]
Thanks. Clean up the blood after, don't want the NSA on our tail!
